static/js/components/HomePage.jsx

```diff
@@ -83,13 +93,12 @@ const HomePage = () => {
     <ResponsiveGridLayout
       className="layout"
       layouts={currentLayouts}
-      breakpoints={{ xlg: 1400, lg: 1150, md: 996, sm: 768, xs: 480 }}
+      breakpoints={{ xlg: 1400, lg: 1150, md: 996, sm: 650, xs: 0 }}
```

Comment:
Should these be defined somewhere and be used throughout the app?

Reply:
Might be a good idea to have defined globally, but I'm not sure. These breakpoint values will probably differ slightly from what we want for the global app because these widths are with respect to viewport width minus the width of the sidebar (except the small ones where the sidebar is closed by default) since that's what is covered by the grid div.